soh/soh/Enhancements/randomizer/3drando/location_access/locacc_castle_town.cpp

```diff
@@ -207,11 +207,13 @@ void AreaTable_Init_CastleTown() {
                   Entrance(RR_THE_MARKET, {[]{return true;}}),
   });
 
-  areaTable[RR_MARKET_BOMBCHU_BOWLING] = Area("Market Bombchu Bowling", "Market Bombchu Bowling", RA_NONE, NO_DAY_NIGHT_CYCLE, {}, {
+  areaTable[RR_MARKET_BOMBCHU_BOWLING] = Area("Market Bombchu Bowling", "Market Bombchu Bowling", RA_NONE, NO_DAY_NIGHT_CYCLE, {
+                  //Events
+                  EventAccess(&CanPlayBowling, {[]{return (randoCtx->GetOption(RSK_BOMBCHUS_IN_LOGIC) && BombchuBag) || (!randoCtx->GetOption(RSK_BOMBCHUS_IN_LOGIC) && BombBag);}}),
```

Comment:
```suggestion
                  EventAccess(&CanPlayBowling, {[]{return randoCtx->GetOption(RSK_BOMBCHUS_IN_LOGIC) ? BombchuBag : BombBag;}}),
```
I don't know if this is preferred with our codestyle but I think it's more simple & intuitive.

Reply:
ternary's are more advanced but IMO it looks cleaner than having the same check so close to itself with a not. it's also probably fractionally more efficient (wouldn't be surprised if the compiler caught it and optimized it though) 

I'll let senior devs chime in for what's preffered

Reply:
As there's no response, I'll just go with the ternary.